docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+sphinx==1.8.5
+sphinx-prompt==1.1.0
+sphinx-tabs==1.1.10
+sphinx-rtd-theme==0.4.3
+git+https://github.com/rtfd/readthedocs-sphinx-search@master
```

Comment:
You are missing `sphinx-notfound-page` here (since you are installing it in conf.py)

Reply:
I have added it.